specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/managedInstances.json

```diff
@@ -361,7 +430,7 @@
           ]
         },
         "dnsZone": {
-          "description": "The Dns Zone that the managed instance is in.",
+          "description": "The Dns Zone taht the managed instance is in.",
```

Comment:
```suggestion
          "description": "The Dns Zone that the managed instance is in.",
```

Reply:
> Not sure if this is generated, but it's reintroducing fixed typos

Yup these were generated, I'll make some fixes in our examples internally and update.
Thanks @nschonni for spotting these.